### ISSUE TITLE
Update fsw_params structs

### ISSUE THREAD

**Issue by tpreynolds** [tpreynolds/uw_hs1_adcs#1]
The init_fsw_params function needs to have all init functions for libraries that have been descoped removed. All dependencies on these libraries need to be resolved.

**Comment by MathiasKerman** [comment]
How should we organize this between descoping plans 1 and 2?

**Comment by tpreynolds** [comment]
good question. Should we assume that descope 1 is contained in descope 2...so that deleting libraries for things like GPS processing & RW control is sufficient?

**Comment by MathiasKerman** [comment]
That sounds like a reasonable thing to do. In that case we would just need to figure out how to separate descope 2 from descope 1 in the case we don't go with 2 while still working on them in parallel
